app/lib/provider_discovery.rb

```diff
@@ -29,7 +29,7 @@ def discover_provider(url, **options)
       end
 
       if format.nil? || format == :xml
-        provider_endpoint ||= html.at_xpath('//link[@type="application/xml+oembed"]')&.attribute('href')&.value
+        provider_endpoint ||= html.at_xpath('//link[@type="text/xml+oembed"]')&.attribute('href')&.value
```

Comment:
Why do you delete support of the correct format `application/xml+oembed`? 

e.g. `//link[@type = "text/xml+oembed" or @type = "application/xml+oembed"]`

Reply:
The specification says nothing about `application/xml+oembed`. `text/xml+oembed` is, on the other hand, mandatory and it is assured that looking for `text/xml+oembed` is enough.

> The type attribute must contain either `application/json+oembed` for JSON responses, or `text/xml+oembed` for XML.

https://oembed.com/

Reply:
Hm. I see.